web-api/openapi/schemas/document.yml

```
@@ -3,6 +3,27 @@ DocumentId:
     $ref: './id.yml#/Id/description'
   $ref: './id.yml#/Id'
 
+SnippetId:
+  description: |-
+    Id of a snippet consisting of the document_id and the in-document id of the given snippet.
+  type: object
+  required:
+    - document_id
+    - snippet_idx
+  properties:
+    document_id:
+      $ref: '#/DocumentId'
+    snippet_idx:
```

🔧 the names have changed
```suggestion
    - sub_id
  properties:
    document_id:
      $ref: '#/DocumentId'
    sub_id:
```